homeassistant/components/zwave_js/fan.py

```diff
@@ -225,3 +236,124 @@ def zwave_speed_to_percentage(self, zwave_speed: int) -> int:
         # the UI handles steps e.g., for a 3-speed fan, you get steps at 33,
         # 67, and 100.
         return round(percentage)
+
+
+class ZwaveThermostatFan(ZWaveBaseEntity, FanEntity):
+    """Representation of a Z-Wave thermostat fan."""
+
+    _fan_mode: ZwaveValue
+    _fan_off: ZwaveValue | None = None
+    _fan_state: ZwaveValue | None = None
+
+    def __init__(
+        self, config_entry: ConfigEntry, client: ZwaveClient, info: ZwaveDiscoveryInfo
+    ) -> None:
+        """Initialize the thermostat fan."""
+        super().__init__(config_entry, client, info)
+
```

Comment:
```suggestion
        super().__init__(config_entry, client, info)

        self._attr_entity_registry_enabled_default = False

```

Reply:
There's also a flag in the discovery schema entry, `entity_registry_enabled_default`.

Reply:
Thanks for the reminder - you should definitely use the flag and let the underlying zwave_js logic handle this instead of overriding it in this way